tzrec/main.py

```diff
@@ -428,6 +444,7 @@ def _train_and_evaluate(
                     _log_train(
                         i_step,
                         losses,
+                        params=optimizer.params,
```

Comment:
add a config for train_config.tensorboard_summaries to control summary what. default is ["learning_rate", "loss"]